packages/expo-analytics-amplitude/src/ExpoAmplitude.web.ts

```diff
@@ -1,5 +1,21 @@
+import amplitude from 'amplitude-js'
+
 export default {
   get name(): string {
     return 'ExpoAmplitude';
   },
-};
+  initialize: (apiKey: string, options?: amplitude.Config) => {
```

Comment:
Add support for `options?: amplitude.Config` on native

Reply:
these are JS only, they are under platform specific settings: https://developers.amplitude.com/?javascript#platform-specific-settings